src/features/script-load.js

```diff
@@ -20,7 +20,7 @@ systemJSPrototype.createScript = function (url) {
   // Only add cross origin for actual cross origin
   // this is because Safari triggers for all
   // - https://bugs.webkit.org/show_bug.cgi?id=171566
-  if (!url.startsWith(baseOrigin + '/'))
+  if (url.indexOf(baseOrigin + '/'))
```

Comment:
@guybedford Shouldn't we be checking for -1 or > 0?   returning 0 (falsy) is in fact true for the check.